Integrations/KennaV2/KennaV2.py

```diff
@@ -0,0 +1,480 @@
+from json import JSONDecodeError
+from typing import List, Tuple, Dict, Any, Optional, Callable
+
+import urllib3
+import demistomock as demisto
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, api_key: str, verify: bool, proxy: bool):
+        header = {
+            'X-Risk-Token': api_key,
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, headers=header)
+
+    def http_request(self, message: str, suffix: str, params: Optional[dict] = None,
+                     data: Optional[dict] = None):
+        return super()._http_request(message, suffix, params=params, json_data=data)
+
+
+def test_module(client: Client, *_):
+    """
+    Performs basic get request to get item samples12
+    """
+    res_vulnerabilities = client._http_request('GET', '/vulnerabilities')
+    res_assets = client._http_request('GET', '/assets')
+
+    if isinstance(res_vulnerabilities.get('vulnerabilities'), list) and isinstance(res_assets.get('assets'), list):
```

Comment:
it's good to have 2 tests.
is it possible(and valid) to have only 1 asset/vuln? - please check and change if necessary.